.azure-pipelines.yml

```diff
@@ -0,0 +1,60 @@
+# This is the configuration file for Azure Pipelines, used to run tests on
+# macOS. Documentation to help understand this setup:
+# https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema
+
+trigger: none # disable builds for branches
+
+jobs:
+- job: macOS
+
+  pool:
+    vmImage: 'macOS-10.13'
+
+  steps:
+  - checkout: self
+    fetchDepth: 50
+    submodules: false
+
+  - script: |
+      echo "Test jobs:"
+      ./wpt test-jobs | while read job; do
+        echo "$job"
+        echo "##vso[task.setvariable variable=run_$job]true";
+      done
+    displayName: 'List test jobs'
+
+  - script: |
+      sudo easy_install pip
+      sudo pip install -U virtualenv
+    displayName: 'Install Python packages'
+    condition: variables.run_wptrunner_infrastructure
+
+  - script: |
+      sudo cp fonts/Ahem.ttf /Library/Fonts
```

Comment:
Is this actually needed? Shouldn't we pass `--install-fonts` to `wpt run` instead?

Reply:
This was after that failed to work once, font installation seems to be racy or something. I guess I could debug it more, how bad do you think this is?

Reply:
It's an issue we've had in buildbot and in gecko CI, so I would like to understand what's going on. If you tried it and it doesn't work out of the box, this is OK for now but we should certainly follow up with the investigations, because `--install-fonts` always works locally on a Mac.

Reply:
Nope, still broken: https://dev.azure.com/foolip/wpt/_build/results?buildId=122&view=logs

I'll file an issue and link it from this file.